package.json

```diff
@@ -18,36 +18,36 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "babel-eslint": "^7.2.3",
-    "broccoli-asset-rev": "^2.5.0",
+    "babel-eslint": "^8.0.1",
+    "broccoli-asset-rev": "^2.6.0",
     "ember-ajax": "3.0.0",
-    "ember-cli": "2.14.1",
-    "ember-cli-app-version": "^3.0.0",
+    "ember-cli": "^2.16.2",
+    "ember-cli-app-version": "^3.1.0",
     "ember-cli-dependency-checker": "^2.0.1",
-    "ember-cli-eslint": "^4.2.0",
-    "ember-cli-htmlbars": "^2.0.2",
-    "ember-cli-htmlbars-inline-precompile": "^0.4.3",
+    "ember-cli-eslint": "^4.2.1",
+    "ember-cli-htmlbars": "^2.0.3",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.2",
     "ember-cli-inject-live-reload": "^1.7.0",
-    "ember-cli-qunit": "^4.0.0",
+    "ember-cli-qunit": "^4.0.2",
     "ember-cli-release": "1.0.0-beta.2",
     "ember-cli-shims": "^1.1.0",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-uglify": "^1.2.0",
-    "ember-data": "^2.14.7",
+    "ember-cli-uglify": "^2.0.0",
+    "ember-data": "^2.16.3",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^2.0.0",
     "ember-load-initializers": "^1.0.0",
-    "ember-resolver": "^4.3.0",
-    "ember-source": "2.14.1",
+    "ember-resolver": "^4.5.0",
+    "ember-source": "^2.16.2",
```

Comment:
For updating ember, use `~2.16.2`, so it will be updated when there are bug fixes